src/rf/apps/core/migrations/0028_allow_null.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
```

Comment:
We haven't been strictly adhering to this rule but it's nice to put model changes and migrations in a single commit separate from application changes.